x/axelarnet/types/msg_call_contract.go

```diff
@@ -44,6 +45,18 @@ func (m CallContractRequest) ValidateBasic() error {
 		return fmt.Errorf("contract address empty")
 	}
 
+	if m.Fee != nil {
+
+		if err := sdk.VerifyAddressFormat(m.Fee.Recipient); err != nil {
+			return sdkerrors.Wrap(sdkerrors.ErrInvalidAddress, sdkerrors.Wrap(err, "fee recipient").Error())
+		}
+
+		if !m.Fee.Amount.IsValid() || !m.Fee.Amount.IsPositive() {
```

Comment:
Ok, I mistook this as just an Int before. Yeah, we need to validate the denom too, sounds good